src/aircraft/FlyByWireA380X.ts

```diff
@@ -0,0 +1,141 @@
+import { AircraftConfigSimType, AircraftFeature, FeatureType } from '../defs'
+import {
+    AircraftConfig,
+    FeatureAddresses,
+    FeatureState,
+    FlapNames,
+    Meta,
+} from '../interface/aircraft'
+
+export default class FlyByWireA380X extends AircraftConfig {
+    meta: Meta = {
+        id: 'flybywire_a380x',
+        name: 'FlyByWire A380X',
+        sim: AircraftConfigSimType.MsFs,
+        enabled: true,
+        priority: 2,
+        author: 'Arthur Parienté <https://github.com/arthurpar06>'
+    }
+
+    features: FeatureAddresses = {
+        [AircraftFeature.BeaconLights]: {
+            LIGHTING_BEACON_0: FeatureType.Int,
+        },
+        [AircraftFeature.LandingLights]: {
+            LIGHTING_LANDING_2: FeatureType.Int,
+        },
+        [AircraftFeature.LogoLights]: {
+            // Lhe logo light isn't available in the FBW A380X at the moment; in the sim, the navigation light currently controls the logo ligh
+            LIGHTING_NAV_0: FeatureType.Int,
+        },
+        [AircraftFeature.NavigationLights]: {
+            LIGHTING_NAV_0: FeatureType.Int,
+        },
+        [AircraftFeature.StrobeLights]: {
+            LIGHTING_STROBE_0: FeatureType.Int,
+        },
+        [AircraftFeature.TaxiLights]: {
+            LIGHTING_LANDING_1: FeatureType.Int,
+        },
+        [AircraftFeature.WingLights]: {
+            LIGHTING_WING_0: FeatureType.Int,
+        },
+        [AircraftFeature.APU]: {
+            A32NX_OVHD_APU_MASTER_SW_PB_IS_ON: FeatureType.Int,
+            A32NX_OVHD_APU_START_SW_PB_IS_ON: FeatureType.Int,
+        },
+        [AircraftFeature.Battery]: {
+            A32NX_OVHD_ELEC_BAT_1_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_2_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_ESS_PB_IS_AUTO: FeatureType.Int,
+            A32NX_OVHD_ELEC_BAT_APU_PB_IS_AUTO: FeatureType.Int,
+        },
+        [AircraftFeature.Seatbelts]: {
+            XMLVAR_SWITCH_OVHD_INTLT_SEATBELT_Position: FeatureType.Int,
+            A32NX_FLAPS_HANDLE_INDEX: FeatureType.Int,
+            A32NX_GEAR_HANDLE_POSITION: FeatureType.Int,
```

Comment:
Understand your logic here but I think the Feature should only return the switch position true / false. This should not be tied to flaps or gear handle position.

Reply:
I wasn't sure either. I was just thinking of reproducing the automatic seat belt behavior

Reply:
I still don't get what you are thinking, or why are you thinking this like a rule... Pilot is meant to set that switch to either auto (or on if he thinks auto will not be enough), but in any way, we should return true because it is what needs to be done by the pilot.

Same logic applies to strobe light too, pilot sets it to auto and it is enough, but in any case it is deemed necessary to turn them on without waiting for the automation (like crossing a runway for a short time), pilots set it to on then revert back to auto.

So, for an aircraft configmap perspective, auto should be considered as `true` like if it is on.

Rest, any deep checks or conditional checks so to say, should go to rules because if you force conditional check in the config file, it may not fit a VA while it is perfectly fine for your ops.

In basic terms, a configmap should not care if the `auto` is working or not 😉